site/homepage/content/release-notes.md

```diff
@@ -11,6 +11,14 @@ title = "Release Notes"
 * The (experimental) CoAP adapter now supports tracking of request processing using
   OpenTracing.
 * A lorawan provider has been added for the loriot network provider.
+* Hono now supports "auto-provisioning" of devices that authenticate with X.509 client certificates. 
+  To enable it, the *auto-provisioning-enabled* property on the trust anchor 
+  must be set to `true` in the tenant configuration. The protocol adapters add the client certificate to the "get" request to the 
```

Comment:
we definitely need a more detailed description of the *auto provisioning* concept which provides information about the overall purpose, constraints, pre-conditions etc.

Reply:
Do you mean in the release notes? Or in the documentation, e.g. user guides or a concept page?

Reply:
I mean in a concept page and maybe also in the user guides ...